client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1523,12 +1524,11 @@ Undocumented.prototype.fetchDns = function( domainName, fn ) {
 	this.wpcom.req.get( '/domains/' + domainName + '/dns', fn );
 };
 
-Undocumented.prototype.addDns = function( domain, record, fn ) {
```

Comment:
@klimeryk: I didn't change any of the actions, only the endpoints, so this shouldn't be an issue.